docs/pyproject.md

```diff
@@ -622,43 +622,48 @@ Poetry is clever enough to detect Python subpackages.
 Thus, you only have to specify the directory where your root package resides.
 {{% /note %}}
 
-### include and exclude
+### exclude and include
```

Comment:
We could add a note about include vs. packages for projects that provide multiple packages. I am not invested in it, though, so it's up to you.

Reply:
Good idea. I added a note.